### ISSUE TITLE
[Bug]: a11y: icon-list - Icons in high contrast mode are not visible

### ISSUE THREAD

**Issue by orsolya-gyovai** [sbb-design-systems/lyne-components#4184]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Components

### Which version of Lyne Components are you using

3.13.0

### Operating system

Windows 11

### Browser / Browser version

Google Chrome Version 142.0.7444.134, Mozilla Firefox 140.4.0

### Input mode

None

### What happened?

**Description**:
In Windows high contrast mode, the check mark icons in the "icon list" are not visible.
The problem only occurs in Chrome; the display of 'icon-list' is correct in Firefox.

<img width="1929" height="924" alt="Image" src="https://github.com/user-attachments/assets/aab04ca2-98dc-47b8-a871-2085ab3a2146" />

**Link**
**The problem only occurs in Chrome**:
https://lyne-storybook.app.sbb.ch/?path=/story/styles-list--icon-list 
https://lyne-storybook.app.sbb.ch/?path=/story/styles-list--icon-list-custom-color 
**The problem occurs in both Chrome and Firefox:**
https://lyne-storybook.app.sbb.ch/?path=/story/styles-list--icon-list-custom-icon 

**Expected behavior**:
The check mark icons should also be visible in Windows high contrast mode.

### Steps to reproduce the issue

**Windows high contrast mode**:
Settings > Accessibility > Contrast themes > Night sky

### Relevant log output

```shell

```
